community/consistency-check/src/main/java/org/neo4j/consistency/checking/full/CountsBuilderDecorator.java

```diff
@@ -162,7 +163,7 @@ public void checkCounts( CountsAccessor counts, final ConsistencyReporter report
             public void visitNodeCount( int labelId, long count )
             {
                 nodeEntries.incrementAndGet();
-                reporter.forCounts( new CountsEntry( nodeKey( labelId ), count ), CHECK_NODE_COUNT );
+                reporter.forCounts( new CountsEntry().initialize( nodeKey( labelId ), count ), CHECK_NODE_COUNT );
```

Comment:
CountsEntry constructor should be called instead directly, like `new CountsEntry( nodeKey( labelId ), count )`
And also in all similar places  in this class